osx/KA-Lite_Uninstall.tool

```diff
@@ -0,0 +1,139 @@
+    # This way, people can enter the username they wish to use
+    # for sudo, and it is more Apple-like.
+
+    osascript -e "do shell script \"/bin/rm -Rf ${REMOVE_FILES_ARRAY[*]}\" with administrator privileges"
+
+    # Verify that the uninstall succeeded by checking whether every file
+    # we meant to remove is actually removed.
+    for file in "${REMOVE_FILES_ARRAY[@]}"; do
+        if [ -e "${file}" ]; then
+            echo "An error must have occurred since a file that was supposed to be"
+            echo "removed still exists: ${file}"
+            echo ""
+            echo "Please try again."
+            key_exit 1
+        fi
+    done
+}
+
+#----------------------------------------------------------------------
+# Script
+#----------------------------------------------------------------------
+# Delete specific location where the kalite is installed.
+# Reliable way for a bash script to get the full path to itself?
+pushd `dirname $0` > /dev/null
+SCRIPTPATH=`pwd`
+popd > /dev/null
+
+# Collect the directories and files to remove
+REMOVE_FILES_ARRAY=()
+append REMOVE_FILES_ARRAY $SCRIPTPATH
+append REMOVE_FILES_ARRAY $HOME_LAUNCH_AGENTS/$KALITE_PLIST
+append REMOVE_FILES_ARRAY $ROOT_LAUNCH_AGENTS/$KALITE_PLIST
+append REMOVE_FILES_ARRAY $KALITE_RESOURCES
+append REMOVE_FILES_ARRAY $KALITE_MONITOR_APP
+append REMOVE_FILES_ARRAY $KALITE_APP
+append REMOVE_FILES_ARRAY $KALITE_USR_BIN_PATH/$KALITE
+append REMOVE_FILES_ARRAY $KALITE_USR_LOCAL_BIN_PATH/$KALITE
+
+# Introduction 
+echo "                                                          "
+echo "   _   __  ___    _     _ _                               "
+echo "  | | / / / _ \  | |   (_) |                              "
+echo "  | |/ / / /_\ \ | |    _| |_ ___                         "
+echo "  |    \ |  _  | | |   | | __/ _ \                        "
+echo "  | |\  \| | | | | |___| | ||  __/                        "
+echo "  \_| \_/\_| |_/ \_____/_|\__\___| Uninstall              "
+echo "                                                          "
+echo "https://learningequality.org/ka-lite/                     "
+echo "                                                          "
+echo "     version 0.16.x                                       "
+echo "                                                          "
+
+
+# Print the files and directories that are to be removed and verify
+# with the user that that is what he/she really wants to do.
+echo "Do you wish to uninstall KA-Lite (y/n)?"
+read user_input
+if [ "$user_input" != "y" ]; then
+    echo "Aborting install. (answer: ${user_input})"
+    key_exit 2
+fi
+
+echo "Do you want to remove the .kalite directory (y/n)?"
+read user_input2
+if [ "$user_input2" == "y" ]; then
+    append REMOVE_FILES_ARRAY "~/.kalite/"
+    echo "Removing .kalite directory (answer: ${user_input2})"
+fi
+
+echo "The following files and directories will be removed:"
+for file in "${REMOVE_FILES_ARRAY[@]}"; do
+    echo "    $file"
+done
+
```

Comment:
@arceduardvincent Can you add a confirmation prompt here?  This will prevent accidental deletions for the user.